src/chapters/git/exercises.rst

```diff
@@ -8,19 +8,117 @@ We will use our new terminal powers to move through the Git exercises.
 #. Commit your local changes using the ``git commit`` procedures.
-#. Add "Hello World!" to the file called ``exercises.txt``.
-#. Commit your local changes following the same steps that you used for step 4.
-#. View and screenshot the result when you use ``git log``. Make note of what you see!
+#. Add ``"Hello World!"`` to the file called ``exercises.txt``.
+#. Commit your local changes following the same steps that you used for step 5.
+#. Run the ``git log`` command. Take a screenshot of the result. Make note of
+   what you see!
 
 Setting up a Github Account
 ---------------------------
 
-For our remote repositories, we will be using `GitHub <https://github.com/>`_. 
+For our remote repositories, we will be using `GitHub <https://github.com/>`__. 
 
 To create your account, follow these steps:
 
 #. Navigate to GitHub's site using the link above.
-#. Sign up for an account on the homepage either by filling out the form or clicking the "Sign Up" button.
+#. Click the *Sign Up* button and follow the on-screen directions.
 #. Once you have an account, you are ready to store your remote work.
+
+Before August 13, 2021, when users pushed changes to a remote repository (or
+pulled new content from it), they were prompted to enter their GitHub username
+and password. However, a username/password combination is not the most
+secure method available. This is especially true for people who reuse the same
+credentials across multiple websites. (You know who you are. Stop doing that!)
+
+After 8/13/2021, GitHub changed its policy in an effort to boost account
+security. Users must now create a *Personal Access Token* or an *SSH key* to
+verify their identity.
+
+Setting up a token for your account is fairly straightforward.
+
+Create a Personal Access Token (PAT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: personal access token
+
+To use HTTPS to push and pull from GitHub, users must create a
+**personal access token**. A PAT takes the place of a password, and the token
+process is considered more secure than a username/password verification.
+
+Once you create your PAT, you will use it instead of your password to perform
+HTTPS Git operations.
+
+.. sourcecode:: bash
+
+   $ git clone https://github.com/username/repo.git
+   Username: your_username
+   Password: your_token
+
+Some users question the need for a PAT, since it looks like another password
+they have to remember. Rather than diving into a lengthy debate and
+justification, we'll focus on the main point: *GitHub requires a PAT or similar
+token*. The platform is incredibly helpful, and we want to use it, so we'll
+follow their advice.
+
+GitHub provides detailed instructions for setting up your PAT, so we will use
+their documentation. Follow steps 1 - 9 for `Creating a Token <https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token>`__
+carefully.
+
+**Tips**:
```

Comment:
glad you were looking out for us PAT newbies with this list of tips.

Reply:
Ha!  This newbie needed the same tips, so that's why I added them here.